docs/build-mvm/setting-up/setting_up.md

```diff
@@ -69,6 +69,6 @@ Now that your account and wallet are setup, select the Fluence network you just
   />
 </div>
 
-Enter the address of your wallet and, if you are using MetaMask, ask for the *tUSDC* token symbol to tbe imported. Then click the "Receive FLT & tUSDC" button. After a short delay, you should see a transaction id and the funds should be in your account. You Blockscout the checkout the transaction.
+Enter the address of your wallet and, if you are using MetaMask, ask for the *tUSDC* token symbol to tbe imported. Then click the "Receive FLT & tUSDC" button. After a short delay, you should see a transaction id and the funds should be in your account. You  the checkout the transaction.
```

Comment:
```suggestion
Enter the address of your wallet and, if you are using MetaMask, ask for the *tUSDC* token symbol to be imported. Then click the "Receive FLT & tUSDC" button. After a short delay, you should see a transaction id and the funds should be in your account. You  the checkout the transaction.
```
I also don't understand the last sentence and it's purpose

Reply:
check the faucet.
anyway, need the links updated asap as users are linked to a dead page.